torch/onnx/_internal/jit_utils.py

```diff
@@ -98,6 +99,49 @@ def aten_op(self, operator: str, *args, overload_name: str = "", **kwargs):
             **kwargs,
         )
 
+    @_beartype.beartype
+    def onnxscript_op(
+        self,
+        onnx_fn,  # TODO(titaiwang): annotate this when onnx-script becomes dependency
+        *raw_args: Union[torch.Tensor, _C.Value],
+        outputs: int = 1,
+        **kwargs,
+    ):
+        """Creates an ONNX operator from onnx-script function, taking "raw_args" as inputs and "kwargs" as attributes.
+
+        onnx-script repository: https://github.com/microsoft/onnx-script
+
+        Args:
+            onnx_fn: ONNXFunction from onnx-script; An example can be found at
+                https://github.com/microsoft/onnx-script#example
+            raw_args: The inputs to the operator; usually provided
+                as arguments to the `symbolic` definition.
+            outputs: The number of outputs this operator returns.
+                By default an operator is assumed to return a single output.
+                If `outputs` is greater than one, this functions returns a tuple
+                of output `Value`, representing each output of the ONNX operator
+                in order.
+            kwargs: The attributes of the ONNX operator, whose keys are named
+                according to the following convention: `alpha_f` indicates
+                the `alpha` attribute with type `f`.  The valid type specifiers are
+                `f` (float), `i` (int), `s` (string) or `t` (Tensor).  An attribute
+                specified with type float accepts either a single float, or a
+                list of floats (e.g., you would say `dims_i` for a `dims` attribute
+                that takes a list of integers).
+
+        Returns:
+            The value representing the single output of this operator (see the `outputs`
+            keyword argument for multi-return nodes).
+        """
+        # TODO(titaiwang): This is using class attributes, and it needs to be updated
+        # if onnx-script makes any change on these.
+        symbolic_name = f"{onnx_fn.opset}::{onnx_fn.opname}"
```

Comment:
Does `onnx_fn.opset` repr shows only domain or does it include version too?

Reply:
I think it would be better to explicitly encode what we want here (instead of making assumptions about `opset` repr (which might change over time).

Reply:
@BowenBao It should be only domain, based on the passed tests in the next PR.
@gramalingam Do you mean hard encode here? If that's the case, the user wouldn't be able to use multiple domains in this kind of usage.

Reply:
I think he was referring to something like `{onnx_fn.opset.domain}`

Reply:
I can turn in a PR in onnx-script to expose this information.

Reply:
Done